### ISSUE TITLE
SFOSC wizard

### ISSUE THREAD

**Issue by Beanow** [sfosc/sfosc#101]
You may have noticed I ran with the wizard idea and created a repo: https://github.com/sfosc/wizard and several issues to start it off.

Based on #78, discussions and ideas as far back as #6's social contract generator, I think this subproject could be a great place to consolidate the overlapping ideas of:

- Researching requirements / mappings of important themes (#84).
- Creating educational SFOSC resources.
- Outputting usable documents for your own community projects.

I'm opening _this_ issue as a place for feedback on the whole of the subproject before there is any strong commitment to it.

In other words: 
If you hate it, now is the time to raise concerns.
If you love it, I'd like to hear it too! But improvement suggestions best go in the new repo.

**Comment by Beanow** [comment]
cc @nothingismagick @NickKellett @themightychris @vsoch
As you've previously helped in work towards this

**Comment by NickKellett** [comment]
Sorry for my delayed response, last few weeks have been unusually busy. I'm in favour of this idea as it should help adoption.
